drivers/interrupt_controller/sam0_eic.c

```diff
@@ -326,12 +327,16 @@ static int sam0_eic_init(struct device *dev)
 {
 	ARG_UNUSED(dev);
 
+	struct device *clk = device_get_binding(DT_ATMEL_SAM0_EIC_0_CLOCK_CONTROLLER);
```

Comment:
always put variable declaration at the beginning of a code block.
(So here, right before ARG_UNUSED)

Reply:
Changed here and a few other places.